bilge-impl/src/bitsize_internal.rs

```diff
@@ -44,24 +45,24 @@ fn generate_struct(struct_data: &ItemStruct, arb_int: &TokenStream) -> TokenStre
 
     let mut fieldless_next_int = 0;
     let mut previous_field_sizes = vec![];
-    let (accessors, (constructor_args, constructor_parts)): (Vec<TokenStream>, (Vec<TokenStream>, Vec<TokenStream>)) = fields.iter()
+    let (accessors, (constructor_args, constructor_parts)): (Vec<TokenStream>, (Vec<TokenStream>, Vec<TokenStream>)) = fields
+        .iter()
         .map(|field| {
             // offset is needed for bit-shifting
             // struct Example { field1: u8, field2: u4, field3: u4 }
             // previous_field_sizes = []     -> unwrap_or_else -> field_offset = 0
             // previous_field_sizes = [8]    -> reduce         -> field_offset = 0 + 8     =  8
             // previous_field_sizes = [8, 4] -> reduce         -> field_offset = 0 + 8 + 4 = 12
-            let field_offset = previous_field_sizes.iter().cloned().reduce(|acc, next| quote!(#acc + #next)).unwrap_or_else(|| quote!(0));
+            let field_offset = previous_field_sizes.iter().cloned()
```

Comment:
e.g. I've reverted the formatting here, I would like it to fill the width I've set and only then wrap to a new line.
I think right now you can only tell rustfmt to wrap all chained calls if the width is overflown.

Reply:
see https://github.com/rust-lang/rustfmt/issues/2263#issuecomment-1648480213